spec/rubocop/cop/rails/save_bang_spec.rb

```diff
@@ -128,6 +128,16 @@
       inspect_source(cop, ['def foo', "object.#{method}", 'end'])
       expect(cop.messages).to be_empty
     end
+
+    it "when using #{method} in || conditional and not last clause" do
```

Comment:
Those 2 tests seems kind of pointless to me - you need some method definition which to check in them...